pkg/templates/manifest.go

```diff
@@ -97,8 +120,19 @@ data:
       - mountPath: /tmp/results
         name: results
         readOnly: false
+{{- if .SSHKey }}
+      - mountPath: /root/.ssh
+        name: sshkey-vol
+{{- end}}
       tolerations:
         - operator: "Exists"
+{{- if .SSHKey }}
+    extra-volumes:
```

Comment:
I think this should be `volumes`.

Reply:
This is for the sonobuoy manifest though so it seems to go through this intermediate `extra-volumes` field that you've created. https://github.com/heptio/sonobuoy/blob/master/docs/plugins.md references this same field for adding more volumes to the plugins.

Reply:
ugh, right you are. sorry, it's been a while since I touched this stuff